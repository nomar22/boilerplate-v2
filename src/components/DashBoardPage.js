import React from 'react';

const DashBoardPage = () => (
    <div >
       Dashboard Page Content
    </div>
);


export default DashBoardPage;