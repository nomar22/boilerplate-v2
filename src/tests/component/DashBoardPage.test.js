import React from   'react';
import {shallow} from 'enzyme';
import DashBoardPage from '../../components/DashBoardPage';



test('Should render expenseDashBoard Page ',()=>{
    const wrapper = shallow(<DashBoardPage />);
    expect(wrapper).toMatchSnapshot();
});


