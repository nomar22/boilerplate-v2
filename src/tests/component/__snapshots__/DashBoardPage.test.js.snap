// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expenseDashBoard Page  1`] = `
<div>
  Dashboard Page Content
</div>
`;
